src/liiteri/api.clj

```diff
@@ -26,41 +27,30 @@
         :middleware [upload/wrap-multipart-params]
         :return schema/File
         (try
-          (response/ok (s3-store/create-file file s3-client db))
+          (response/ok (file-store/create-file file storage-engine db))
           (finally
-            (.delete (:tempfile file)))))
+            (io/delete-file (:tempfile file) true))))
 
       (api/GET "/files/metadata" []
         :summary "Get metadata for one or more files"
         :query-params [key :- (api/describe [s/Str] "Key of the file")]
         :return [schema/File]
-        (let [metadata (file-store/get-metadata key db)]
+        (let [metadata (file-metadata-store/get-metadata key db)]
           (if (> (count metadata) 0)
             (response/ok metadata)
```

Comment:
This doesn't apply, the list is actually for each different file asked for with the `key` parameter.